util/tagrepos.py

```diff
@@ -0,0 +1,46 @@
+from github import Github, GithubException
+import sys
+import csv
+
+if len(sys.argv) <> 3:
+    print "Please specify both username and github password."
+    print "Usage: tagrepos.py <username> <password>"
+    print "Takes CSV formatted stream as input following the format: "
+    print "github-reponame, sha1, tag"
+    print "Example:"
+    print "jbosstools/jbosstools-hibernate, a1f287bf825a891901b41739f6a4bbaafc70bcdd, jbosstools-4.2.1.Alpha23"
+    print
+    print "Useful when combined with buildinfo.json: "
+    print "cat buildinfo.json | python buildinfo_to_tags.py -n <tagname> | python tagrepos.py <taggeruser> <secret>" 
```

Comment:
I link this filename better: "buildinfo_to_tags" is more easy to read than "buildinfototags" which looks like "build in foto tags". However, you need to then rename the file in this PR: https://github.com/maxandersen/jbosstools-build-ci/blob/tagtools/util/buildinfototags.py

At any rate, please be consitent w/ your usage instructions and filenames.